examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -119,10 +137,9 @@ typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
-    current<UsedParticleCurrentSolver>
-    #if(PARAM_IONIZATION==1)
-    ,ionizer<particles::ionization::BSI<PIC_Electrons> >
-    #endif
+    current<UsedParticleCurrentSolver>, 
+    atomicNumbers<Helium>,
+    ionizer<particles::ionization::BSI<PIC_Electrons> >
```

Comment:
If someone does not want ionization, they can just remove the ionizer flag. But I don't know if this is too inconvenient right now for the current state of ionization.